tests/src/main/java/org/hibernate/beanvalidation/tck/tests/messageinterpolation/ExpressionLanguageMessageInterpolationTest.java

```diff
@@ -241,4 +253,33 @@ public boolean isValid(Continent continent, ConstraintValidatorContext constrain
 			}
 		}
 	}
+
+	private static class TestWrapper {
+
+		private final TestLogic testLogic;
+		private final Locale originalLocale;
+		private final Locale testLocale;
+
+		public TestWrapper(TestLogic testLogic, Locale testLocale) {
+			this.testLogic = testLogic;
+			this.originalLocale = Locale.getDefault();
+			this.testLocale = testLocale;
+		}
+
+		public void doTest() {
+			Locale.setDefault( testLocale );
+			testLogic.test();
+			Locale.setDefault( originalLocale );
+		}
+
+		public static void test(TestLogic testLogic) {
+			new TestWrapper( testLogic, Locale.ENGLISH ).doTest();
```

Comment:
This is weird. You push the ENGLISH locale as the testLocale every time, while it might not be indeed. But if it's not, you use `Locale.setDefault` in the `TestLogic`.

I would rather add a second `test` method taking an additional locale parameter and use this locale for the test. It looks like it's what you wanted to do but decided otherwise in the way :).

I would even rename the method `testWithEnglishLocale` and `testWithSpecificLocale` to be more explicit.

Or am I missing something?